rs/registry/canister/canister/canister.rs

```diff
@@ -882,6 +882,24 @@ fn get_node_providers_monthly_xdr_rewards_() -> Result<NodeProvidersMonthlyXdrRe
     registry().get_node_providers_monthly_xdr_rewards()
 }
 
+#[export_name = "canister_query get_api_boundary_node_ids"]
+fn get_api_boundary_node_ids() {
+    over(
+        candid_one,
+        |()| -> Result<Vec<ApiBoundaryNodeIdRecord>, String> { get_api_boundary_node_ids_() },
```

Comment:
I think this could be simplified to

```suggestion
        get_api_boundary_node_ids_,
```